src/code/z_player_lib.c

```diff
@@ -1536,6 +1506,10 @@ void Player_PostLimbDrawGameplay(PlayState* play, s32 limbIndex, Gfx** dList, Ve
             Matrix_Get(&this->shieldMf);
         } else if ((this->rightHandType == PLAYER_MODELTYPE_RH_BOW_SLINGSHOT) ||
                    (this->rightHandType == PLAYER_MODELTYPE_RH_BOW_SLINGSHOT_2)) {
+            static BowSlingshotStringData sBowSlingshotStringData[] = {
+                { gLinkAdultBowStringDL, { 0.0f, -360.4f, 0.0f } },        // Bow
+                { gLinkChildSlingshotStringDL, { 606.0f, 236.0f, 0.0f } }, // Slingshot
+            };
```

Comment:
sorry, should line breaks go after all of these too? There's a couple more below

Reply:
this case and all the others still have more variables being newly declared under it. the linebreak goes after the declarations are done (even if those declarations are also assigning)